.eslintrc.js

```diff
@@ -4,10 +4,11 @@ module.exports = {
     project: 'tsconfig.json',
     sourceType: 'module',
   },
-  plugins: ['@typescript-eslint/eslint-plugin'],
+  plugins: ['@typescript-eslint/eslint-plugin', '@darraghor/nestjs-typed'],
```

Comment:
move this part out in favor of https://github.com/Visual-Regression-Tracker/backend/pull/233/files

Reply:
I will rebase on top of master once there is updates to master...